src/Locale/ja_JP/cake.po

```diff
@@ -0,0 +1,285 @@
+msgid "{0,number,#,###.##} KB"
+msgstr "{0,number,#,###.##}KB"
+
+#: I18n/Number.php:91
+msgid "{0,number,#,###.##} MB"
+msgstr "{0,number,#,###.##}MB"
+
+#: I18n/Number.php:93
+msgid "{0,number,#,###.##} GB"
+msgstr "{0,number,#,###.##}GB"
+
+#: I18n/Number.php:95
+msgid "{0,number,#,###.##} TB"
+msgstr "{0,number,#,###.##}TB"
+
+#: I18n/Number.php:87
+msgid "{0,number,integer} Byte"
+msgid_plural "{0,number,integer} Bytes"
+msgstr[0] "{0,number,integer}バイト"
+msgstr[1] "{0,number,integer}バイト"
+
+#: I18n/RelativeTimeFormatter.php:80
+msgid "{0} from now"
+msgstr "今から{0}後"
+
+#: I18n/RelativeTimeFormatter.php:80
+msgid "{0} ago"
+msgstr "今から{0}前"
+
+#: I18n/RelativeTimeFormatter.php:82
+msgid "{0} after"
+msgstr "{0}後"
+
+#: I18n/RelativeTimeFormatter.php:82
+msgid "{0} before"
+msgstr "{0}前"
+
+#: I18n/RelativeTimeFormatter.php:113
+msgid "just now"
+msgstr "たった今"
+
+#: I18n/RelativeTimeFormatter.php:150
+msgid "about a second ago"
+msgstr "約1秒前"
+
+#: I18n/RelativeTimeFormatter.php:151
+msgid "about a minute ago"
+msgstr "約1分前"
+
+#: I18n/RelativeTimeFormatter.php:152
+msgid "about an hour ago"
+msgstr "約1時間前"
+
+#: I18n/RelativeTimeFormatter.php:153;326
+msgid "about a day ago"
+msgstr "約1日前"
+
+#: I18n/RelativeTimeFormatter.php:154;327
+msgid "about a week ago"
+msgstr "約1週間前"
+
+#: I18n/RelativeTimeFormatter.php:155;329
+msgid "about a year ago"
+msgstr "約1年前"
+
+#: I18n/RelativeTimeFormatter.php:165
+msgid "in about a second"
+msgstr "約1秒後"
+
+#: I18n/RelativeTimeFormatter.php:166
+msgid "in about a minute"
+msgstr "約1分後"
+
+#: I18n/RelativeTimeFormatter.php:167
+msgid "in about an hour"
+msgstr "約1時間後"
+
+#: I18n/RelativeTimeFormatter.php:168;339
+msgid "in about a day"
+msgstr "約1日後"
+
+#: I18n/RelativeTimeFormatter.php:169;340
+msgid "in about a week"
+msgstr "約1週間後"
+
+#: I18n/RelativeTimeFormatter.php:170;342
+msgid "in about a year"
+msgstr "約1年後"
+
+#: I18n/RelativeTimeFormatter.php:362
+msgid "%s ago"
+msgstr "%s前"
+
+#: I18n/RelativeTimeFormatter.php:363
+msgid "on %s"
+msgstr "%s"
+
+#: I18n/RelativeTimeFormatter.php:47;125;310
+msgid "{0} year"
+msgid_plural "{0} years"
+msgstr[0] "{0}年"
+msgstr[1] "{0}年"
+
+#: I18n/RelativeTimeFormatter.php:51;128;313
+msgid "{0} month"
+msgid_plural "{0} months"
+msgstr[0] "{0}か月"
+msgstr[1] "{0}か月"
+
+#: I18n/RelativeTimeFormatter.php:57;131;316
+msgid "{0} week"
+msgid_plural "{0} weeks"
+msgstr[0] "{0}週間"
+msgstr[1] "{0}週間"
+
+#: I18n/RelativeTimeFormatter.php:59;134;319
+msgid "{0} day"
+msgid_plural "{0} days"
+msgstr[0] "{0}日"
+msgstr[1] "{0}日"
+
+#: I18n/RelativeTimeFormatter.php:64;137
+msgid "{0} hour"
+msgid_plural "{0} hours"
+msgstr[0] "{0}時間"
+msgstr[1] "{0}時間"
+
+#: I18n/RelativeTimeFormatter.php:68;140
+msgid "{0} minute"
+msgid_plural "{0} minutes"
+msgstr[0] "{0}分"
+msgstr[1] "{0}分"
+
+#: I18n/RelativeTimeFormatter.php:72;143
+msgid "{0} second"
+msgid_plural "{0} seconds"
+msgstr[0] "{0}秒"
+msgstr[1] "{0}秒"
+
+#: I18n/RelativeTimeFormatter.php:298
+msgid "today"
```

Comment:
Thanks. By the way the msgid 'today' was not extracted by i18n shell. But it exists in [RelativeTimeFormatter.php](https://github.com/cakephp/cakephp/blob/df6b8483893439ae4527e678fc6b963e76f43dae/src/I18n/RelativeTimeFormatter.php#L298) certainly. May be a bug?